get-started/index.md

```diff
@@ -9,15 +9,33 @@ layout: posts
 Cryostat {{ site.data.versions.cryostat.version }}
 
 ## [Installing Cryostat Operator](#installing-cryostat-operator)
-Coming soon to [OperatorHub](https://operatorhub.io/). In the meantime, you can install
-the Cryostat Operator using kubectl, or by deploying the bundle image with Operator SDK.
-
+To install the Cryostat Operator, you can choose one of the following installation options: install via OperatorHub (recommended), install with kubectl, or install by deploying the bundle image with Operator SDK. All of the installation methods require cert-manager as a prerequisite.
 ### Install cert-manager
 The Cryostat Operator requires [cert-manager](https://cert-manager.io/) to run.
 If not already installed in your cluster, please 
 [install](https://cert-manager.io/docs/installation/) it using your preferred method.
 Once installed, proceed with one of the Cryostat Operator installation options below.
 
+### Install via OperatorHub (recommended)
+See below for a summary of the installation steps from the Cryostat Operator page on [OperatorHub](https://operatorhub.io/cryostat-operator). For more details, visit [Installing the Cryostat Operator from OperatorHub](https://developers.redhat.com/articles/2022/01/20/install-cryostat-operator-kubernetes-operatorhubio#). 
+
+1. Install the Operator Lifecycle Manager (OLM):
+```
+$ curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.20.0/install.sh | bash -s v0.20.0
+```
+2. Verify the installation was successful by confirming all pods are `READY`:
+```
+$ kubectl get pods -n olm
+```
+3. Install the Cryostat Operator:
+```
+$ kubectl create -f https://operatorhub.io/install/cryostat-operator.yaml
+```
+4. Check the status of the operator deployment. When the operator phase reads `Succeeded`, you are ready to set up and deploy Cryostat:
+```
+$ kubectl get csv -n my-cryostat-operator -w
+```
+
 ### Install with kubectl
```

Comment:
> I'm not sure if we want to keep the "Install with kubectl" option as well. If we do, I would place the OperatorHub.io option above it as our preferred method.

Revisiting this, what are the benefits to either keeping or removing the "install with kubectl" section?

Reply:
I think we should have it documented *somewhere* since it's probably the best way for us to point users to pre-release upstream versions that might contain changes and fixes that need testing, or if a user accepts the possible stability issues of using a pre-release version from upstream before we productize it downstream. I don't know if we want to advertise that front-and-center like we do the OperatorHub installation method, which is definitely the most end-user, consumer friendly, since it will only contain what should be our well-tested release versions.

Maybe this should be moved to something else like an operator GitHub Wiki page? Or maybe we want another separate page on our cryostat.io homepage that has alternate installation methods, and we can add a footnote reference to that instead of calling out kubectl as an installation method here on the Getting Started?

Reply:
> Or maybe we want another separate page on our cryostat.io homepage that has alternate installation methods,

I like how this idea would simplify the Get Started page and also keep users on the same website. If we do move the kubectl installation steps to another page, we'd also want to the move the "install bundle with Operator SDK" steps as well for consistency.

Reply:
> > Or maybe we want another separate page on our cryostat.io homepage that has alternate installation methods,
> 
> I like how this idea would simplify the Get Started page and also keep users on the same website. If we do move the kubectl installation steps to another page, we'd also want to the move the "install bundle with Operator SDK" steps as well for consistency.

This all sounds good to me. We should advertise OperatorHub.io as the primary way to consume upstream releases, and other methods can be moved/hidden elsewhere.